### PR TITLE
{data}[fosscuda/2018b] bsddb3 v6.2.6

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/bsddb3/bsddb3-6.2.6-fosscuda-2018b-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Oracle/Sleepycat C API for the Database Environment, Database, Cursor,
+Log Cursor, Sequence and Transaction objects."""
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['42d621f4037425afcb16b67d5600c4556271a071a9a7f7f2c2b1ba65bc582d05']
+
+osdependencies = [('libdb-dev')]
```
Comment (boegel) [inline easybuild/easyconfigs/b/bsddb3/bsddb3-6.2.6-fosscuda-2018b-Python-2.7.15.eb]: @akesandgren Any change of resolving this via EasyBuild instead?

File: easybuild/easyconfigs/b/bsddb3/bsddb3-6.2.6-fosscuda-2018b-Python-2.7.15.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+dependencies = [('Python', '2.7.15')]
+
+use_pip = True
+download_dep_fail = True
+
+preinstallopts = 'unset LIBS && '
```
Comment (boegel) [inline easybuild/easyconfigs/b/bsddb3/bsddb3-6.2.6-fosscuda-2018b-Python-2.7.15.eb]: @akesandgren Why is this required exactly? Maybe add a comment above this line to explain?

File: easybuild/easyconfigs/b/bsddb3/bsddb3-6.2.6-fosscuda-2018b-Python-2.7.15.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/b/bsddb3/bsddb3-6.2.6-fosscuda-2018b-Python-2.7.15.eb]: Technically possible, but I'm currently too lazy to deal with it.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/62768b0c19f23fb02f7764c27c30430d for a full test report.

File: easybuild/easyconfigs/b/bsddb3/bsddb3-6.2.6-fosscuda-2018b-Python-2.7.15.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/b/bsddb3/bsddb3-6.2.6-fosscuda-2018b-Python-2.7.15.eb]: Well, at least that's honest... :P

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3134.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/1f10e17cf389b6a67d45264e89159c32 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/5e1f54e7af514abd35e55dd87c9f1dd5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2077.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/54eb810c0ac4ee95d7e4f05b74fc9e6b for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!
